app/assets/javascripts/hyrax/fileupload.js

```diff
@@ -5,4 +5,7 @@ Blacklight.onLoad(function() {
   var options = {};
   $('#fileupload').hyraxUploader(options);
   $('#fileuploadlogo').hyraxUploader({downloadTemplateId: 'logo-template-download'});
+  $("#new_generic_work, #new_batch_upload_item, .edit_generic_work").on('submit', function(event) {
```

Comment:
`generic_work` isn't something we can depend on.  This is going to be whatever was generated.  e.g. book, photograph, etc.

Furthermore, shouldn't this be part of the hyraxUploader module? https://github.com/samvera/hyrax/blob/8283950efad5e0d1c8fc8e304895399506550c59/app/assets/javascripts/hyrax/uploader.js#L23